parcels/particleset.py

```diff
@@ -191,7 +200,10 @@ def ArrayClass_init(self, *args, **kwargs):
             self._repeatkwargs = kwargs
             self._repeatkwargs.pop("partition_function", None)
 
-        ngrids = fieldset.gridset.size
+        if type(fieldset) == UXFieldSet:
+            ngrids = 1
```

Comment:
Here, we explicitly set ngrids to 1. However, with the idea that the `fieldset` will store a list of either XArray or UXArray datasets, we will want to set ngrids to the length of the list of datasets.